anyway/app_and_db.py

```diff
@@ -1,5 +1,27 @@
 from anyway import utilities
 from flask_sqlalchemy import SQLAlchemy
 
+from anyway.backend_constants import BE_CONST
+from anyway.config import SERVER_ENV
+
 app = utilities.init_flask()
 db = SQLAlchemy(app)
+
+
+def get_cors_config() -> dict:
+    cors_site_list = BE_CONST.ANYWAY_CORS_SITE_LIST_PROD
+    if SERVER_ENV == "dev":
```

Comment:
What is the difference between `SERVER_ENV` and `FLASK_ENV` ?

Reply:
The `SERVER_ENV` is used to determine if we are in `https://dev.anyway.co.il/` server or in `https://www.anyway.co.il/` server
and `FLASK_ENV` is used to determine if we are development state (of the backend server) or not.

In the `https://dev.anyway.co.il/` server `SERVER_ENV=dev` and `FLASK_ENV` is not set,
In the `https://anyway.co.il/` server `SERVER_ENV=prod` and `FLASK_ENV` is not set,
on the local system `SERVER_ENV` is not set and `FLASK_ENV=development`.


*`https://dev.anyway.co.il/` is for the FE team development.

Reply:
From the name `SERVER_ENV` I thought it is something that signals development versus production and testing. Maybe it is worthwhile to change the name so it indicates it is referring to a specific hostname that the backend resides on.

Reply:
I wasn't able to come up with a better name so i added a comment to better explain what `SERVER_ENV` means.

Reply:
I agree, perhaps it's better to add the documentation @BarVolunteering added as a comment in another location? 
Do you think we need to open a new doc? @ziv17 @BarVolunteering  what do you think?

Reply:
Maybe a page about the environment variables ?

Reply:
Regarding documentation, I prefer placing it in the code where the variable is defined.
In general, both `SERVER_ENV` and `FLASK_ENV` are configured values. One is set at installation time, another I would like us to be able to set also at each invocation.